src/irr/asset/CImageLoaderPNG.cpp

```diff
@@ -189,10 +193,19 @@ asset::IAsset* CImageLoaderPng::loadAsset(io::IReadFile* _file, const asset::IAs
 	else
 	{
 		double image_gamma;
-		if (png_get_gAMA(png_ptr, info_ptr, &image_gamma))
+
+		if (png_get_gAMA(png_ptr, info_ptr, &image_gamma)) {
+
+		
 			png_set_gamma(png_ptr, screen_gamma, image_gamma);
-		else
+		}
+
+		else {
+			
+			os::Printer::log(std::to_wstring(image_gamma), ELL_DEBUG);
```

Comment:
nice but add a human readabke string beforehand, because someone will just get a random number out of nowhere in the console and not know what it means